task/01-strings-tasks.js

```diff
@@ -20,7 +18,7 @@
  *   '',  'bb'  => 'bb'
  */
 function concatenateStrings(value1, value2) {
-  throw new Error('Not implemented');
+  return value1 + value2;
```

Comment:
Please take a look at the Airbnb js style guide.
https://github.com/airbnb/javascript#strings--line-length
template string is a more preferable way for building a string from parts since it looks more obvious

Reply:
fixed